### ISSUE TITLE
Add www subdomain to Janitor

### ISSUE THREAD

**Issue by arshdkhn1** [JanitorTechnology/janitor#228]
![screenshot 2018-01-14 00 23 45](https://user-images.githubusercontent.com/16819670/34909106-3a8221e2-f8c1-11e7-8291-7b6b824c96d9.png)

**Comment by jankeromnes** [comment]
Indeed, thanks for reporting this issue! I'm not against adding a `www` sub-domain (if only to redirect to without the `www.` prefix), but I'm curious about what benefits this would yield.

Do some services or tools expect a `www.` prefix, or is it simply for convenience because some people might be used to that?

**Comment by arshdkhn1** [comment]
Simply for convenience, I dont have any solid opinion. :sweat_smile:
